docs/source/upgrade/upgrade-guide-from-monitoring-3.x-to-monitoring-4.y.rst

```diff
@@ -0,0 +1,266 @@
+==============================================================
+Upgrade Guide - Scylla Monitoring 3.x to Scylla Monitoring 4.y
+==============================================================
+
+This document is a step by step procedure for upgrading `Scylla Monitoring Stack </operating-scylla/monitoring/3.0>`_ from version 3.x to 3.y, for example, between 3.0 to 3.0.1.
+
+Upgrade Procedure
+=================
+
+We recommend installing the new release next to the old one, running both in parallel, and making sure it is working as expected before uninstalling the old version.
+
+Change to the directory you want to install the new Monitoring stack.
+Download the latest release:
+You can download the .zip or the .tar.
+
+Install 4.y (The new version)
+-----------------------------
+
+.. code-block:: bash
+
+                wget -L https://github.com/scylladb/scylla-monitoring/archive/scylla-monitoring-4.y.zip
+                unzip scylla-monitoring-4.y.zip
+                cd scylla-monitoring-scylla-monitoring-4.y/
+
+Replace “y” with the new minor release number, for example, 4.0.0.zip
+
+Setting the server's files
+--------------------------
+
+Copy the ``scylla_servers.yml`` and ``scylla_manager_servers.yml`` from the version that is already installed.
+
+.. code-block:: bash
+
+                cp /path/to/monitoring/3.x/prometheus/scylla_servers.yml prometheus/
+                cp /path/to/monitoring/3.x/prometheus/scylla_manager_servers.yml.yml prometheus/
+
+Validate the new version is running the correct version
+-------------------------------------------------------
+
+Starting from Scylla-Monitoring version 2.2, you can run:
+
+.. code-block:: bash
+
+                ./start-all.sh --version
+
+To validate the Scylla-Monitoring version.
+
+
+Validate the version installed correctly
+----------------------------------------
+
+To validate that the Monitoring stack starts correctly, first in parallel to the current (3.x) stack.
```

Comment:
I don't quite understand "first in parallel to the current (3.x) stack".
Should it be "first in parallel to the current (3.x) stack"?